app/controllers/chargeback_report_controller.rb

```diff
@@ -5,71 +5,46 @@ class ChargebackReportController < ApplicationController
   after_action :set_session_data
 
   include Mixins::SavedReportPaging
+  include Mixins::GenericListMixin
   include Mixins::GenericSessionMixin
+  include Mixins::GenericShowMixin
   include Mixins::BreadcrumbsMixin
 
   def self.table_name
-    @table_name ||= "chargeback_report"
+    @table_name ||= "miq_report_result"
   end
 
-  def x_show
-    @explorer = true
+  def self.model
+    MiqReportResult
```

Comment:
See above.

Reply:
I do need this defined in order to get list view to work, there is no `ChragebackReport` model

Reply:
As far as I can tell, I think the last place this would be used is the toolbar chooser.
So if the toolbar works...

Reply:
I am leaning towards having a `ChargebackReport` STI model. Probably not in this PR...cc @Fryguy

Reply:
> I am leaning towards having a `ChargebackReport` STI model. Probably not in this PR...cc @Fryguy

cc @gtanzillo